resources/uploads/models/upload_writable.yml

```diff
@@ -6,8 +6,9 @@ required:
 
 properties:
   campaignId:
-    $ref: "../../campaigns/attributes/cmp_id.yml"
-    description: Associated Campaign ID
-    example: cmp_1933ad629bae1408
+    allOf:
+      - $ref: "../../campaigns/attributes/cmp_id.yml"
+      - description: Associated Campaign ID
+      - example: cmp_1933ad629bae1408
```

Comment:
I'll note that the `example` unfortunately isn't showing up in the docs but tis a nonblocking comment :P Thanks for fixing spectral~